### PR TITLE
feat: support target-tracking autoscaling

### PR DESCRIPTION
**Pull request by francesco-racciatti**
# Description

This PR adds a [TargetTracking](https://docs.aws.amazon.com/autoscaling/application/userguide/application-auto-scaling-target-tracking.html)-based mechanism to support the automatic horizontal scaling of the orchestrator.

Note that:
 - autoscaling is disabled by default,
 - no additional resources are deployed when autoscaling is disabled.

### CODE REVIEW

**Review by pgcrooks-sysdig — APPROVED** [review]
LGTM

**Comment by francesco-racciatti** [comment]
@pgcrooks-sysdig yes, will do 🙂

File: README.md [hunk 1]
```diff
@@ -94,6 +97,11 @@ No modules.
...
 | <a name="input_http_proxy_configuration_ca_certificate"></a> [http\_proxy\_configuration.ca\_certificate](#http\_proxy\_configuration.ca\_certificate)                           | Configures the SSL connection to the HTTP proxy - The path to the Ca certificate to use in the SSL connection to the HTTP proxy                                                                                                                     | `string`       | `""`                                                                                        |    no    |
+| <a name="input_autoscaling_target_metric"></a> [autoscaling.target\_metric](#autoscaling.target\_metric)                                                                         | Sets the TargetTracking metric, can be either `ECSServiceAverageCPUUtilization` or `CSServiceAverageMemoryUtilization`                                                                                                                              | `string`       | `""`                                                                                        |    no    |
```
Comment (pgcrooks-sysdig) [inline README.md:100]:
```suggestion
| <a name="input_autoscaling_target_metric"></a> [autoscaling.target\_metric](#autoscaling.target\_metric)                                                                         | Sets the TargetTracking metric, can be either `ECSServiceAverageCPUUtilization` or `ECSServiceAverageMemoryUtilization`                                                                                                                              | `string`       | `""`                                                                                        |    no    |
```

**Review by pgcrooks-sysdig — CHANGES_REQUESTED** [review]
1 NIT

**Review by pgcrooks-sysdig — APPROVED** [review]
LGTM

**Pull request merged**